src/NCPkgTable.h

```diff
@@ -133,13 +137,41 @@ class NCPkgTableSort : public NCTableSortStrategyBase {
 		YTableItem *secondItem = dynamic_cast<YTableItem*> (second->origItem() );
 		NCPkgTableTag *firstTag = static_cast<NCPkgTableTag *>( firstItem->cell(0) );
 		NCPkgTableTag *secondTag = static_cast<NCPkgTableTag *>( secondItem->cell(0) );
-		
+
 		return firstTag->getDataPointer()->installSize() <
 		       secondTag->getDataPointer()->installSize();
 	    }
 
     };
 
+    class CompareName
+    {
+    public:
+	CompareName( int uiCol)
+	    : _uiCol(uiCol)
+	    {}
+
+	bool operator() ( NCTableLine * first,
+			  NCTableLine * second
+			  ) const
+	    {
+                std::wstring w1 = first->GetCol( _uiCol )->Label().getText().begin()->str();
+                std::wstring w2 = second->GetCol( _uiCol )->Label().getText().begin()->str();
+
+                // It is safe to use collate unaware wscasecmp() here because package names
+                // are 7 bit ASCII only. Better yet, we don't even want this to be sorted
+                // by locale specific rules: "ch" for example would be sorted after "h" in
+                // Czech which in the context of package names (which are English) would be
+                // plain wrong.
+                int result = wcscasecmp( w1.data(), w2.data() );
```

Comment:
It's included in NCurses.h from libyui-ncurses. NCurses.h is the mother of all libyui-ncurses include files, i.e. it's included everywhere. But anyway that's nothing new since that other functor class where I stole this from uses wcscollate() from the same header wchar.h.

But thanks for noticing!